src/types/type_wrappers.rs

```diff
@@ -50,6 +51,29 @@ impl ComplexType {
     pub fn get_second_support_type(&self) -> Option<ComplexType> {
         self.support_type_2.as_ref().map(|f| *f.clone())
     }
+
+    #[napi]
+    pub fn remap_list_support_type(&self, new_subtype: &ComplexType) -> ComplexType {
+        ComplexType::one_support(self.base_type, Some(new_subtype.clone()))
+    }
+
+    #[napi]
+    pub fn remap_map_support_type(
+        &self,
+        key_new_subtype: &ComplexType,
+        val_new_subtype: &ComplexType,
+    ) -> ComplexType {
+        ComplexType::two_support(
+            self.base_type,
+            Some(key_new_subtype.clone()),
+            Some(val_new_subtype.clone()),
+        )
+    }
+
+    #[napi]
+    pub fn get_empty() -> ComplexType {
+        ComplexType::simple_type(CqlType::Custom)
+    }
```

Comment:
:wrench: missing docstrings for `pub` functions.

Reply:
In general, functions on Rust-NodeJS boundary must be well-documented, because this is where the complicated things happen and we need clarity.